providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/FlagdOptions.java

```diff
@@ -140,5 +129,22 @@ public FlagdOptionsBuilder withGlobalTelemetry(final boolean b) {
             }
             return this;
         }
+
+        void prebuild() {
+            if (resolverType == null) {
+                resolverType = fromValueProvider(System::getenv);
+            }
+
```

Comment:
nit - I think this is because we cannot access `resolverType` inside the static class. But we can access `propertyName$value` here (see `Small print` section) [1]

if so in the line 144 , we can simply use `if (resolverType$value.equals(Config.Resolver.RPC))` :) 

[1] - https://projectlombok.org/features/Builder

Reply:
thanks for your hint @Kavindu-Dodan. I tried to access the `resolverType` as you proposed but it didn't work for me :(

Reply:
Let's ignore this suggestion. The fine print says ` This is an implementation detail; do not write code that interacts with these fields` which IMO is not favourable :)